spec/support/jasmine-browser.json

```diff
@@ -0,0 +1,16 @@
+{
+  "srcDir": "public/assets",
+  "srcFiles": [
+    "govuk-admin-template-*.js",
+    "application-*.js"
+  ],
+  "cssFiles": [
+    "application-*.css"
+  ],
```

Comment:
These look to be the right files to test but I notice they don't match those of the previous configuration: https://github.com/alphagov/manuals-publisher/blob/c106827f4fbb7d9b46c442025ab8bad3ac9f8376/spec/javascripts/support/jasmine.yml#L13-L30

I think the previous version was configured a bit wrong though - perhaps someone trying to get themselves out of a bind. It'd be good if we could state in the commit message why we've configured it differently